### PR TITLE
added doc.txt to ds18_sensor test

### PR DESCRIPTION
**Pull request by xkevin190**
### Contribution description

the doc.txt was added to the ds18 sensor test module where it is explained how to use it and how to execute it

### CODE REVIEW

File: tests/ds18/doc.txt [hunk 1]
```diff
@@ -0,0 +1,33 @@
+/**
+@defgroup ds18_test DS18 test mdoule
```
Comment (luisan00) [inline tests/ds18/doc.txt:2]: what? :smile:

File: tests/ds18/doc.txt [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+
+```c
+void test_init_ds18(void) {
+     int err = init_temperature_sensor(your_gpio);
+
+     TEST_ASSERT_EQUAL_INT(0, error);
+}
+```
+
+To test the code from python:
```
Comment (luisan00) [inline tests/ds18/doc.txt:19]: same

File: tests/ds18/doc.txt [hunk 3]
```diff
@@ -0,0 +1,33 @@
...
+
+```c
+make flash test
+
+```
+
+to test only C code"
```
Comment (luisan00) [inline tests/ds18/doc.txt:26]: same

**Review by luisan00 — CHANGES_REQUESTED** [review]
reviewed

File: tests/ds18/doc.txt [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+@ingroup     test_group
+@{
+
+
+## Test DS18 sensor
+
+Here unit tests are performed on the ds18_sensor module.
+To perform a successful test you must connect the ds18 temperature sensor to your board.
+
+you must also add the gipio where your board will be connected in the test init function in the main.c file
```
Comment (eduazocar) [inline tests/ds18/doc.txt:12]: gpio

**Review by eduazocar — COMMENTED** [review]
Fix some misspellings words

File: tests/ds18/doc.txt [hunk 5]
```diff
@@ -0,0 +1,30 @@
+/**
+@defgroup ds18_test DS18_SENSOR test mdoule
```
Comment (eduazocar) [inline tests/ds18/doc.txt:2]: Still having the same mistake reviewed by @luisan00

Comment (eduazocar) [reply tests/ds18/doc.txt:2]: Refers to test module

File: tests/storage/doc.txt [hunk 6]
```diff
@@ -0,0 +1,31 @@
...
+```
+
+To test the code in interactive mode:
+
+```c
+make flash term
+
+```
+@}
+ */
```
Comment (eduazocar) [inline tests/storage/doc.txt:31]: Why this document is includes?

**Review by eduazocar — COMMENTED** [review]
Let's go, only a simple corrections, and that it's all

File: tests/ds18/doc.txt [hunk 7]
```diff
@@ -0,0 +1,30 @@
+/**
+@defgroup ds18_test DS18_SENSOR test module
```
Comment (luisan00) [inline tests/ds18/doc.txt:2]: why capital words? and why underscore between words?

**Review by luisan00 — CHANGES_REQUESTED** [review]
one more please

File: tests/storage/doc.txt (hunk 6 above)
Comment (luisan00) [reply tests/storage/doc.txt:31]: looks a copy paste issue xD

**Review by eduazocar — APPROVED** [review]
Good job :+1:

**Review by luisan00 — APPROVED** [review]
👍

**Pull request merged**